README.md

```diff
@@ -1,25 +1,21 @@
-# Backend starter code
+# Metadata Generator code
 
-This repository is to be used as a starter for Koa based Node.js applications written in TypeScript and using MongoDB. It includes a range of nice packages and configurations. It also has a stub for user login via Telegram, Facebook and Google. Enjoy!
+This repistory is used to be metadata generator for derivatives on selcred.xyz
```

Comment:
`This repository is used to be a metadata generator for derivatives on sealcred.xyz`

Reply:
fixed, ty